salt/motd/motd.jinja

```diff
@@ -1,7 +1,8 @@
-Welcome!
+Welcome to SUSE CaaS Platform!
 
 Machine ID: {{ salt['grains.get']('machine_id', 'ERROR') }}
 Hostname: {{ salt['grains.get']('nodename', 'ERROR') }}
+Container Runtime: {{ salt.caasp_cri.cri_name() }}
```

Comment:
Maybe we can add yet another helper `cri_human_name` or something similar so we can output on the motd something like: "Docker Open Source Container Engine".

Reply:
I think a pillar value would be better, so we could do `{{ pillar['cri'][salt.caasp_cri.cri_name()]['description'] }}`